pkg/controller/command/command.go

```diff
@@ -150,16 +150,31 @@ func (c *Cmd) GetPublicKey(w io.Writer, _ io.Reader) error {
 }
 
 // CreateLeaf creates MerkleTreeLeaf.
-func CreateLeaf(timestamp uint64, credential []byte) *MerkleTreeLeaf {
+func CreateLeaf(timestamp uint64, credential []byte) (*MerkleTreeLeaf, error) {
+	vc, err := verifiable.ParseCredential(credential,
+		verifiable.WithDisabledProofCheck(),
```

Comment:
Where are we validating the proofs - is it done at a a higher layer?

Reply:
yes